_config.yml

```diff
@@ -303,66 +363,60 @@ defaults:
       grand_parent: SQLFlow
       nav_order: 3
   - scope:
-      path: "elasticdl/doc/data_preparation_design.md"
+      path: "elasticdl/docs/designs/data_preparation.md"
     values:
       nav_exclude: false
       parent: ElasticDL 
       nav_order: 1
   - scope:
-      path: "elasticdl/doc/distributed_embedding_layer_design.md"
+      path: "elasticdl/docs/designs/distributed_embedding_layer.md"
     values:
       nav_exclude: false
       parent: ElasticDL 
       nav_order: 2
   - scope:
-      path: "elasticdl/doc/tutorials/elasticdl_cloud.md"
+      path: "elasticdl/docs/tutorials/elasticdl_cloud.md"
```

Comment:
We can use this file instead of elasticdl_cloud.md
https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/tutorials/get_started.md.

And we can put this file in the first order of ElasticDL.